scripts/calculate-average-block-times.ts

```diff
@@ -6,8 +6,11 @@ import { CHAINS } from '../src';
 const BLOCK_LOOKBACK = 400_000;
 
 async function calculateAverageBlockTimes(): Promise<void> {
+  const specifiedChain = CHAINS.find(chain => chain.alias === process.env.CHAIN);
```

Comment:
A command line arg would probably be better imo (i.e. `yarn providers:time --chain-alias=xyz`), but this works too.

Reply:
Yeah it's not ideal but I'm in a rush